mandatory/4-tax.js

```diff
@@ -15,9 +17,18 @@ function calculateSalesTax() {}
   Write a function that adds tax to a number, and then transforms the total into the format £0.00
 
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
+
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(num) {
+   return `£${calculateSalesTax(num).toFixed(2)}`;
+}
+
+//CAN ALSO WRITE THIS WAY:
+// function addTaxAndFormatCurrency(num) {
+//   var totalPrice= `£${calculateSalesTax(num).toFixed(2)}`;
```

Comment:
Yes, as discussed in the previous comment, the first is more concise, but sometimes breaking your code down into stages and declaring variables with good names helps to make your code more understandable. It's up to you which you prefer.